index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href = "style.css">
+    <title>Document</title>
+
+</head>
+<body>
+    <h1 class="Intro">Hello Microverse!</h1>
+</body>
```

Comment:
- [ ] i would suggest you to use `header tag` use html file that includes a header with the text "Hello Microverse!". Use h1 tag for that,this will make your code readable.

Reply:
@ademibowale I don't fully understand what you mean here. Do you mean that the text "Hello Microverse" should be the the header of the webpage and not in the body as I had done it?